packages/material-ui/src/darkScrollbar/index.ts

```diff
@@ -6,28 +6,28 @@ const scrollBar = {
 };
 
 export default function darkScrollbar(options = scrollBar) {
-  return {
-    scrollbarColor: `${options.thumb} ${options.track}`,
-    '&::-webkit-scrollbar, & *::-webkit-scrollbar': {
-      backgroundColor: options.track,
-    },
-    '&::-webkit-scrollbar-thumb, & *::-webkit-scrollbar-thumb': {
-      borderRadius: 8,
-      backgroundColor: options.thumb,
-      minHeight: 24,
-      border: `3px solid ${options.track}`,
-    },
-    '&::-webkit-scrollbar-thumb:focus, & *::-webkit-scrollbar-thumb:focus': {
-      backgroundColor: options.active,
-    },
-    '&::-webkit-scrollbar-thumb:active, & *::-webkit-scrollbar-thumb:active': {
-      backgroundColor: options.active,
-    },
-    '&::-webkit-scrollbar-thumb:hover, & *::-webkit-scrollbar-thumb:hover': {
-      backgroundColor: options.active,
-    },
-    '&::-webkit-scrollbar-corner, & *::-webkit-scrollbar-corner': {
-      backgroundColor: options.track,
-    },
-  };
+  return `
```

Comment:
This may break if it's used with JS objects :( Not sure what is the best approach here. If we use string templates for the style overrides for the CssBaseline, I think this change is mandatory.. cc @oliviertassinari  Maybe we should also consider using custom `deepmerge` function for the `CssBaseline` overrides where we would concat array values, instead of blindly replacing them.